fractal_server/app/api/__init__.py

```diff
@@ -17,6 +18,9 @@
 router_default = APIRouter()
 router_v1 = APIRouter()
 
+router_v1.include_router(
+    monitoring_router, prefix="/monitoring", tags=["Monitoring"]
```

Comment:
Choose right name (monitoring? anything better? TBD)

Reply:
Let's include `router_monitoring` in `fractal_server/main.py`, and use it here so that the endpoints are `/monitoring/..` (without `api/v1`)